packages/vite/package.json

```diff
@@ -4,8 +4,8 @@
   "type": "module",
   "exports": {
     ".": {
-      "default": "./index.mjs",
-      "types": "./index.d.ts"
+      "default": "./build/index.js",
```

Comment:
could we go with the more standard dist?

Reply:
sure 🤷 fine either way 👍

Reply:
fixed 🎉